src/labkey/Query.ts

```diff
@@ -57,4 +60,36 @@ export {
     URL_COLUMN_PREFIX,
     validateQuery,
     Visualization
+}
+
+export interface GetQueryMetadataOptions {
+    containerPath?: string
```

Comment:
For the `getQueryMetadataOptions` interface let's make the following changes:

* At minimum `schemaName` and `queryName` are required properties.
* Documentation needs to be added for each property.
* Alphabetical declaration of interface properties.